wiki/People/Beatmap_Nominators/en.md

```diff
@@ -8,137 +8,153 @@
 [Yf]: /wiki/shared/Insane-f.png "Yes"
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
-Beatmap Nomination Group
+Beatmap Nominators
 ========================
 
-The **Beatmap Nomination Group**, more commonly referred to as the BNG, is a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Individual members are commonly referred to as Beatmap Nominators, or BNs.
-This group has undergone many changes recently.
-As of 22. August 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums.
-The most recent change was on 01. February 2015 when the team was changed from *Beatmap Appreciation Team* to *Beatmap Nominator Group*.
+The **Beatmap Nominators** are a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Individual members are sometimes referred to by the acronym *BN*.
+
+Formerly, they were considered as part of the volunteer moderation team. On the 22nd of August, 2014, they were split from their moderation duties to focus completely on the modding and ranking process.
+
+The Beatmap Nominators is split into two tiers based on their level of proven expertise. These two tiers have different privileges and responsibilities.
+
+At the moment, **only the osu!standard Beatmap Nominators are split according to tiers.** Other gamemodes will receive their tier splits at a later date.
 
 Roles and Responsibilities
 --------------------------
 
 ![The Nominator Badge](Nominator.png "The Nominator Badge")
 
-Members of the **Beatmap Nomination Group** are responsible primarily for [modding](/wiki/Modding) the [pending beatmaps](/wiki/Beatmaps) into the ranking cycle.
+The **Beatmap Nominators** are responsible primarily for [modding](/wiki/Modding) the [pending beatmaps](/wiki/Beatmaps) into the ranking cycle.
+
 BNs are the people to go to for any modding or advice requests about anything mapping or modding related.
 
+In modes that have not yet had a tier split (osu!taiko, osu!catch, osu!mania), the BN are responsible for both bubbling and qualifying maps.
+
+Otherwise:
+
+Tier 1 or 'beginner' BN can only bubble maps. They may not qualify them.
```

Comment:
make the two sentences into one?
`Tier 1 or 'beginner' BN can only bubble maps, but may not qualify them.`